java2typescript-jackson/README.md

```diff
@@ -68,7 +68,14 @@ export type MyEnum = "VAL1" | "VAl2" | "VAL3";
 ```
 
 This will keep strong-typing for the values, while allowing for string literal values of enum properties. Take a look at
-the [test that turns on this preference](src/test/java/java2typescript/jackson/module/WriterPreferencesTest.java#L67)
+the [test that turns on this preference](src/test/java/java2typescript/jackson/module/WriterPreferencesTest.java#L67).
+
+There is an additional option you can set on the writer which, if enabled, generates typescript constants which mirror the enum values. To enable this, do the following:
+
+```Java
+mWriter.preferences.useEnumPattern();
+mWriter.preferences.createConstantsForStringLiteralTypeEnums();
```

Comment:
Done. I chose to overload the method so as to keep with backwards compatibility, plus i think logically an opt-in feature like this shouldn't require an "opt-out" `false` being passed. I hope that is okay.